runtime/compiler/env/j9method.cpp

```diff
@@ -1364,11 +1370,14 @@ TR_ResolvedRelocatableJ9Method::TR_ResolvedRelocatableJ9Method(TR_OpaqueMethodBl
             SVM_ASSERT_ALREADY_VALIDATED(svm, aMethod);
             SVM_ASSERT_ALREADY_VALIDATED(svm, containingClass());
             }
-         else if (owner) // JITServer: in baseline, if owner doesn't exist then comp doesn't exist, so thi case is not possible
-            // but in JITClient comp is initialized before creating resolved method for compilee, so need this guard.
+#if defined(JITSERVER_SUPPORT)
```

Comment:
This ifdef changes the original definition of the constructor that exists in the mainline.
Some restructuring of the code is required.